week-2/G-conditionals-2/exercise-4.js

```diff
@@ -8,8 +8,13 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
-
+function containsCode(sentence) {
+  if (sentence.includes("code")) {
```

Comment:
You can simplify this to `return sentence.includes("code")`. You don't need an `if`.

Reply:
Ok,Thanks